packages/hash/api/src/graphql/resolvers/pages/pageProperties.ts

```diff
@@ -0,0 +1,61 @@
+import { ApolloError } from "apollo-server-express";
+import { FieldNode } from "graphql";
+import {
+  Page,
+  UnresolvedGQLPage,
+  UnresolvedGQLUnknownEntity,
+} from "../../../model";
+import {
+  PageProperties as GQLPageProperties,
+  Resolver,
+} from "../../apiTypes.gen";
+import { GraphQLContext } from "../../context";
+
+/**
+ * IMPORTANT NOTE: this is a temporary field resolver and will be deprecated
+ * once API consumers have been refactored to stop accessing `properties.contents`
+ * of a Page, and access these using the `linkGroups` or `contents` fields instead
+ */
```

Comment:
Great to have this context. 
What are your thoughts on adding a `@deprecated` jsdoc annotation here? I don't feel strongly about this, but it might signal to any dev that this is meant to be phased out

Reply:
Sounds good, have done so here: https://github.com/hashintel/hash/pull/303/commits/01239e8db89f24c499ec1dc4e77acdc81c8f0687

Just to note though these resolvers are only being used `src/graphql/resolvers/index.ts`, but I agree this makes it even clearer these resolvers won't stick around 👍